src/docs/framework-integration/angular.md

```diff
@@ -39,211 +43,199 @@ top-most-directory/
+    angularOutputTarget({
+      componentCorePackage: 'your-stencil-library-package-name',
+      directivesProxyFile: '../angular-workspace/projects/component-library/src/lib/stencil-generated/components.ts',
+      directivesArrayFile: '../angular-workspace/projects/component-library/src/lib/stencil-generated/index.ts',
     }),
-    {
-      type: 'dist',
-      esmLoaderPath: '../loader',
-    },
-    {
-      type: 'dist-custom-elements',
-    },
   ],
 };
 ```
 
-Once you have added this and have `directivesProxyFile` pointing to the correct directory of your Angular component library, you can build Stencil. 
+> The `directivesProxyFile` is the relative path to the file that will be generated with all of the Angular component wrappers. You will replace the file path to match your project's structure and respective names. You can generate any file name instead of `components.ts`.
 
-`npm run build`
+> The `directivesArrayFile` is the relative path to the file that will be generated with a constant of all the Angular component wrappers. This constant can be used to easily declare and export all the wrappers.
 
-Or
+You can now build your Stencil component library to generate the component wrappers.
 
-`yarn build`
+```bash
+# Build the library and wrappers
+npm run build
+# or if using yarn
+yarn run build
+```
 
-Once the build is complete, you will see new files in your Angular component library's directory!
+If the build is successful, you will now have contents in the file specified in `directivesProxyFile` and `directivesArrayFile`.
 
+You can now finally import and export the generated component wrappers for your component library. For example, in your library's main Angular module:
 
-```
-top-most-directory/
-├── stencil-library/
-│   ├── stencil.config.js
-│   └── src/components
-└── angular-workspace/
-    └── projects/
-        └── component-library/
-            └── src/
-                ├── lib/
-                │   └── stencil-generated/components.ts
-                └── public-api.ts
-```
+```ts
+import { DIRECTIVES } from './stencil-generated';
 
-### Add the components to your Angular project’s entry file (public-api.ts)
+@NgModule({
+  declarations: [...DIRECTIVES],
+  exports: [...DIRECTIVES],
+})
+export class ExampleLibraryModule {}
+```
 
-In order to make the generated files available to your Angular component library and it’s consumers, you’ll need to export everything from within your entry file - commonly the `public-api.ts` file. To do this, you’ll write:
+Any components that are included in the `exports` array should additionally be exported in your main entry point (either `public-api.ts` or `index.ts`). Skipping this step will lead to Angular Ivy errors when building for production.
 
-```tsx
-export * from './lib/stencil-generated/components';
+```ts
+export { DIRECTIVES } from './stencil-generated';
```

Comment:
Should be `'./lib/stencil-generated'` right? But even then, for some reason it doesn't work. `export * from './lib/stencil-generated/components';` does!

Reply:
Nevermind it does work with `'./lib/stencil-generated'`